root/static/scripts/release-editor/trackParser.js

```diff
@@ -485,17 +485,19 @@ trackParser.customDelimiterError = debounce(function () {
 })
 
 function optionCookie(name, defaultValue, checkbox=true) {
-    var existingValue = getCookie(name);
-
-    if (checkbox) {
-      var observable = ko.observable(
-          defaultValue ? existingValue !== "false" : existingValue === "true"
-      );
-    } else {
-      var observable = ko.observable(
-          existingValue ? existingValue : defaultValue
-      );
-    }
+    let existingValue = getCookie(name);
+
+    let observable = checkbox  
```

Comment:
```suggestion
    const observable = checkbox  
```

Reply:
Hi @mwiencek, thanks for your review i will change it as soon as possible